__tests__/app.test.js

```diff
@@ -44,6 +44,41 @@ describe("GET /api/", () => {
     });
   });
 
+  describe("GET /api/articles/:article_id", () => {
+    test("GET:200 - responds with a 200 status code and an article matching the supplied article id", () => {
+      return request(app)
+        .get("/api/articles/5")
+        .expect(200)
+        .then(({_body : {article}}) => {
+          expect(article.length).toBe(1);
+          expect(typeof article[0].article_id).toBe("number");
```

Comment:
We know what the article ID is going to be here, so rather than expect any number, we can expect the specific one 👍

Reply:
Aye, cheers :)